frontend/packages/operator-lifecycle-manager/src/dev-catalog.ts

```diff
@@ -4,6 +4,22 @@ import { ClusterServiceVersionKind } from './types';
 import { referenceForProvidedAPI, providedAPIsFor } from './components';
 import * as operatorLogo from './operator.svg';
 
+const isInternal = (crd: { name: string }): boolean => {
+  const internalOpListString = _.get(
+    crd,
+    ['csv', 'metadata', 'annotations', 'operators.operatorframework.io/internal-objects'],
+    '[]',
+  );
+  try {
+    const internalOpList = internalOpListString.slice(1, -1).split(','); // JSON.parse fails or returns improper structure
```

Comment:
Proper JSON uses double quotes for serialization. Therefore the correct JSON value of the annotation will be: `["foo","bar"]`.
That is why `JSON.parse` is failing for you.

Reply:
Using JSON.parse in try and console.error in catch